alephclient/tasks/crawldir.py

```
@@ -2,24 +2,30 @@
 import threading
 from queue import Queue
 from pathlib import Path
+from typing import Optional, Dict
 
 from alephclient import settings
+from alephclient.api import AlephAPI
 from alephclient.errors import AlephException
 from alephclient.util import backoff
 
 log = logging.getLogger(__name__)
 
 
-def _get_foreign_id(root_path, path):
+def _get_foreign_id(root_path: Path, path: Path) -> Optional[str]:
     if path == root_path:
         if path.is_dir():
-            return
+            # NOTE: Raise exception
+            return None
         return path.name
     if root_path in path.parents:
         return str(path.relative_to(root_path))
+    # NOTE: Raise exception
+    return None
```

I think we should raise an exception there.